debian.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Does this file have to be at the root level? It looks like a source asset for assembling the linux package, could it go in a `src` or `installers/linux` directory?

Reply:
doesn't need to be there. Will move it.